tokio/src/net/windows/named_pipe.rs

```diff
@@ -1973,9 +1970,10 @@ impl ServerOptions {
     /// use std::{io, os::windows::prelude::AsRawHandle, ptr};
     //
     /// use tokio::net::windows::named_pipe::ServerOptions;
-    /// use winapi::{
-    ///     shared::winerror::ERROR_SUCCESS,
-    ///     um::{accctrl::SE_KERNEL_OBJECT, aclapi::SetSecurityInfo, winnt::DACL_SECURITY_INFORMATION},
+    /// use windows_sys::{
+    ///     Win32::Foundation::ERROR_SUCCESS,
```

Comment:
It's not obvious to me where I indicate the crate and feature dependencies needed for the docs to compile. Any ideas?

Reply:
This seems to be the only thing holding up the build.

Reply:
You could try this: (
```toml
[target.'cfg(any(windows,docsrs))'.dependencies.windows-sys]
version = "0.42.0"
optional = true
```
But it depends on whether windows-sys will compile on unix.

Reply:
Thanks, I'll give this a go. It does compile on Unix, provided you don't try to link.

Reply:
Does this mean that we don't need to use URLs for windows-sys links?

Reply:
Hopefully! I'm having a hard time testing this locally. `cargo doc -p tokio --open --all-features` doesn't seem to compile the examples on Windows.

Reply:
So `cargo test --doc` works to compile the examples - I threw in a typo to confirm - but I'm not seeing the same errors so I'm guessing some features are not getting enabled on the CI doc build...

Reply:
Try building docs like this:
```
RUSTDOCFLAGS="--cfg docsrs" RUSTFLAGS="--cfg docsrs" cargo +nightly doc --all-features
```

Reply:
You can also see the documentation generated by CI by pressing the "netlify/tokio-rs/deploy-preview" CI check.